ts/features/fci/hooks/useFciAbortSignatureFlow.tsx

```diff
@@ -44,6 +47,7 @@ export const useFciAbortSignatureFlow = () => {
       rightButton={{
         ...errorButtonProps(() => {
           dismiss();
+          trackFciUserExit(route.name);
           dispatch(fciEndRequest());
```

Comment:
Same as above about "tracking" and "dispatching". What do you think to have a standard also for "dismissing" and "dispatching"?